bucket/ravendb.json

```diff
@@ -19,6 +19,7 @@
             "hash": "8e65f887a677e78a92e57faf71363c17a1b2bf54b0c490f5b2618b597117dd99"
         }
     },
+    "pre_install": "if(!(Test-Path $persist_dir\\settings.json)) {Set-Content $dir\\settings.json -Value $null}",
```

Comment:
- Unqoted path
- Do not use $null in set-content
    - Use new-item or set it value
- Always use encoding in set-content